dist/metacom.js

```diff
@@ -19,21 +19,9 @@ class MetacomError extends Error {
   }
 }
 
-class MetacomInterface {
+class MetacomInterface extends EventEmitter {
   constructor() {
-    this._events = new Map();
-  }
-
-  on(name, fn) {
-    const event = this._events.get(name);
-    if (event) event.add(fn);
-    else this._events.set(name, new Set([fn]));
-  }
-
-  emit(name, ...args) {
-    const event = this._events.get(name);
-    if (!event) return;
-    for (const fn of event.values()) fn(...args);
+    super();
```

Comment:
In this case we do not need constructor at all

Reply:
MetacomInterface make sens for any features that may need in future

Reply:
I added a method for adding an Interface, if we need to have some Schemas scaffolding to a specific method, so we can just write any functionality for this with `MetacomInterface` in the same way, that we have in `impress` `Interfaces` and `Procedure` classes. There will be no need to make `Metacom.load` heavy.